examples/direct.py

```diff
@@ -0,0 +1,80 @@
+# fmt: off
+import sys
+from typing import BinaryIO
+from itertools import repeat
+from collections.abc import Iterator
+
+import boto3
+import pyarrow.fs
+
+import aryn_sdk.partition
+import sycamore
+from sycamore.transforms.embed import SentenceTransformerEmbedder
+from sycamore.transforms.sketcher import Sketcher
+from sycamore.connectors.duckdb.duckdb_writer import (
+    DuckDBWriterClientParams,
+    DuckDBWriterTargetParams,
+    DuckDBWriter,
+)
+
+###############################################################################
+
+def iterInputs(inputs: list[str], aws_sess = None) -> Iterator[BinaryIO]:
```

Comment:
You can replace all of this with
docs = BinaryScan(paths=inputs).local_source() 
once https://github.com/aryn-ai/sycamore/pull/712/files is in.